the-name-card/src/components/index.ts

```diff
@@ -0,0 +1,8 @@
+export { default as Landing } from './Landing/Landing';
```

Comment:
nit: personally i prefer using named exports 

e.g.
```
// Section.tsx
const Section ...
...

export { Section };
```

```
// components/index.ts

export * from './Section/Section'
```

Reply:
right that is good suggestion